### PR TITLE
feat(#12): add floating action button with menu on profile screen

### PR DESCRIPTION
**Pull request by R-Mielamud**

### CODE REVIEW

**Comment by R-Mielamud** [comment]
Looks like this:
Знімок екрана 2022-02-10 о 16 29 12

File: package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "react-native": "0.67.2",
     "react-native-config": "^1.4.5",
     "react-native-encrypted-storage": "^4.0.2",
+    "react-native-floating-action": "^1.22.0",
```
Comment (AleksandrTermenzhy) [inline package.json]: I don't think we need extra library here. Please, take a look: https://callstack.github.io/react-native-paper/fab.html

Comment (R-Mielamud) [reply package.json]: This library has a "magic" actions property, which creates a beautiful menu

Comment (AleksandrTermenzhy) [reply package.json]: I'm not sure which magic you're talking about but check this out:
IMG_20220210_205148

Isn't this a magic?)

Comment (R-Mielamud) [reply package.json]: Fixed in 5b3ca09

**Review by AleksandrTermenzhy — APPROVED** [review]

File: src/screens/Profile/index.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
```
Comment (liza-veis) [inline src/screens/Profile/index.tsx]: Remove this file and move it's logic to profile.tsx, also rename Profile folder to lowercase

File: src/screens/Profile/index.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import { View } from 'react-native';
+import { FAB } from 'react-native-paper';
+import { SafeAreaView } from 'react-native-safe-area-context';
+import addActions from './add-actions';
```
Comment (liza-veis) [inline src/screens/Profile/index.tsx]: Add blank line after libraries imports

File: src/screens/Profile/index.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+  return (
+    <SafeAreaView style={styles.fullHeight}>
+      <View style={styles.fullHeight}>
+        <FAB.Group
+          open={addMenuOpen}
+          visible={true}
+          icon="plus"
+          actions={addActions(handleItemPress)}
+          onStateChange={({ open }) => setAddMenuOpen(open)}
```
Comment (liza-veis) [inline src/screens/Profile/index.tsx]: let's put it in a separate callback like handleMenuStateChange

File: src/screens/Profile/add-actions.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
+const addActions = (select: (name: string) => void) => {
```
Comment (liza-veis) [inline src/screens/Profile/add-actions.tsx]: Change this file extension to .ts

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Comment by Liza-Veis** [comment]
Now yarn.lock has missing dependencies, please reinstall dependencies and commit changes

File: src/screens/profile/profile.tsx [hunk 6]
```diff
@@ -1,16 +1,52 @@
...
+  const handleItemPress = (name?: string) => {
+    if (!name) {
+      return;
+    }
+
+    // addFunctions[name]();
+  };
+
+  const handleMenuStateChange = ({ open }: { open: boolean }) =>
+    setAddMenuOpen(open);
```
Comment (liza-veis) [inline src/screens/profile/profile.tsx]:
```suggestion
  const handleMenuStateChange = ({ open }: { open: boolean }) => {
    setAddMenuOpen(open);
  }
```

**Review by liza-veis — APPROVED** [review]

**Pull request merged**